packages/server/src/fhir/operations/expand.ts

```diff
@@ -266,32 +277,70 @@ async function computeExpansion(
     } else {
       await includeInExpansion(include, expansion, codeSystem, params);
     }
-
-    if (expansion.length > (count ?? MAX_EXPANSION_SIZE)) {
-      // Return partial expansion
-      break;
-    }
   }
 
+  await addExpandJob(valueSet, codeSystemCache); // Precompute expansion in the background for faster future searches
   return expansion;
 }
 
+async function isExpansionPrecomputed(valueSet: ValueSet): Promise<boolean> {
+  const result = await new SelectQuery('ValueSet_Membership')
+    .column('coding')
+    .where('valueSet', '=', valueSet.id)
+    .limit(1)
+    .execute(getDatabasePool(DatabaseMode.READER));
+  return result.length >= 1;
+}
+
+async function getPrecomputedExpansion(
+  valueSet: ValueSet,
+  params: ValueSetExpandParameters
+): Promise<ValueSetExpansionContains[]> {
+  const query = new SelectQuery('Coding').column('code').column('display');
+  const joinAlias = query.getNextJoinAlias();
+  query
+    .innerJoin(
+      'ValueSet_Membership',
+      joinAlias,
+      new Condition(new Column('Coding', 'id'), '=', new Column(joinAlias, 'coding'))
```

Comment:
I'd want to measure it, but including `valueSet` in the `ON` clause might be able to take advantage of the index better?  Or maybe postgres is smart enough to figure that out so they're equivalent?